jwst/group_scale/group_scale.py

```diff
@@ -22,34 +22,26 @@ def do_correction(input_model):
 
     Parameters
     ----------
-    input_model: data model object
-        science data to be corrected
-
-    Returns
-    -------
-    output_model: data model object
-        rescaled science data
-
+    model: data model object
+        science data to be corrected. Model is modified in place
+        but also returned.
```

Comment:
"but also returned" is no longer accurate?

Reply:
bah! forgot to remove that...